### PR TITLE
 updated website docs and Demo

### PR DESCRIPTION
**Pull request by EsmaShr**
Can you please merge? ASAP thank you!

### CODE REVIEW

**Review by dsymiller — APPROVED** [review]

**Review by mmeigs — APPROVED** [review]
Looks amazing! Great additions.

File: server.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { React, ReactDomServer } from './deps.ts';
 import App from './client/app.tsx';
 import { staticFileMiddleware } from './staticFileMiddleware.ts';
 
-const PORT = 3000;
+const PORT = 3333;
```
Comment (travismfrank) [inline server.tsx:6]: Hey ya'll, is this correct?  The dockerfile exposes port 3000, I think this is causing a 502 error

Comment (dsymiller) [reply server.tsx:6]: good catch! we are fixing that and a few other things and will be making another pull request shortly.

Comment (travismfrank) [reply server.tsx:6]: Awesome!  When you make the new PR, request me for review and I'll take a sec to look it over and approve. :)

Comment (dsymiller) [reply server.tsx:6]: copy! will do

**Pull request merged**